src/components/modules/document/common/Upload/modules/Video/modules/Recorder/Recorder.view.vue

```diff
@@ -82,6 +91,12 @@ export default {
     };
   },
 
+  methods: {
```

Comment:
if you are using compose api, no need to create `methods` and etc fields.
all in `setup` method

Reply:
done